File: frontendapp/lib/dtos/BookingDTO.dart
class BookingDTO{
  final int userId;
  final int showtimeId;
  final double totalPrice;
  final String paymentMethod;
  final String paymentStatus;

  BookingDTO({
    required this.userId,
    required this.showtimeId,
    required this.totalPrice,
    required this.paymentMethod,
    required this.paymentStatus,
  });

  Map<String, dynamic> toJson() {
    return {
      'userid': userId,
      'showtimeid': showtimeId,
      'totalprice': totalPrice,
      'paymentmethod': paymentMethod,
      'paymentstatus': paymentStatus,
    };
  }
}

File: webadmin/src/app/dtos/booking.dto.ts
export class BookingDTO {
    id: number;
    userId: number;
    showTimeId: number;
    bookingdate: string;
    totalprice: number;
    paymentmethod: string;
    paymentstatus: string;
    isactive: boolean;
    constructor(data: any) {
        this.id = data.id;
        this.userId = data.userId;
        this.showTimeId = data.showTimeId;
        this.bookingdate = data.bookingdate;
        this.totalprice = data.totalprice;
        this.paymentmethod = data.paymentmethod;
        this.paymentstatus = data.paymentstatus;
        this.isactive = data.isactive;
    };
}